Template.php

```diff
@@ -444,6 +444,19 @@ public function add_if_new($param_name, $value) {
   }
 
   protected function get_identifiers_from_url() {
+    if ($this->blank('url') && !$this->blank('website')) {  // No URL, but a website
+      $url = trim($this->get('website'));
+      if (strtolower(substr( $url, 0, 4 )) !== "http" ) {
+        $url = "https://" . $url; // Try it with http
+      }
+      if (filter_var($url, FILTER_VALIDATE_URL,FILTER_FLAG_HOST_REQUIRED |  FILTER_FLAG_PATH_REQUIRED ) ) { // is a valid http(s) URL with a full hostname and path of some type
+        $this->set('url',$url);
+        $this->forget('website');
```

Comment:
Rather than using "set" and "forget", it would be better to rename the existing parameter; this will preserve its position in the citation and its formatting/spacing.

Reply:
will do